internal/common/utils_test.go

```diff
@@ -85,3 +85,20 @@ func TestGetUniqueOrigin(t *testing.T) {
 		t.Errorf("origin1: %d should <= origin2: %d", origin1, origin2)
 	}
 }
+
+func TestFilterQueryParams(t *testing.T) {
+	query := "test1=test1&ds-name=name&test2=tset2&ds-id=id"
```

Comment:
use test struct to test multiple query strings
example: https://github.com/edgexfoundry/edgex-go/blob/master/internal/core/metadata/rest_addressable_test.go
your test should cover:
1. "ds-" in the middle of name
2. "ds-" prefix in the value
3. "ds-" in the end of name
4.  multiple the same name with "ds-" prefix